pyproject.toml

```diff
@@ -2,4 +2,4 @@
 target-version = ['py36', 'py37']
 skip-string-normalization = true
 line-length = 79
-exclude = 'vendored|_vispy'
+exclude = 'vendored|\_vispy'
```

Comment:
this should go in a separate PR and also include other commonly ignored files e.g. `build`, `dist`, `.ipynb_checkpoints`, etc.

Reply:
As above, I'm not sure why its `\_vispy` instead of `_vispy` I don't remember doing that and can remove it.

I think black might already be ignoring files in our `.gitignore`?

Reply:
removed from this PR